src/mumble/EchoCancelOption.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "EchoCancelOption.h"
+
+#include <QObject>
+
+const std::vector< EchoCancelOption > &EchoCancelOption::getOptions() {
+	// Note that we have to create this list latily (at the time it is needed for the first time) in order
+	// to make sure that there actually is a translator installed that can translate these Strings. If we
+	// create these objects too early (e.g. by making them a static variable in global namespace), the Strings
+	// will not get translated.
+
+	// Please strictly follow the order of the EchoCancelOptionID enum when adding items to this array.
```

Comment:
I see this is taken over, but I think we should make this a hard MUST. The method now specifies that it is indexable as such, so this MUST remain true.

```suggestion
	// The item order MUST follow the EchoCancelOptionID enum.
```

```suggestion
	// The item order MUST follow the EchoCancelOptionID enum so they remain accessible by index by enum value.
```